Makefile

```diff
@@ -9,6 +9,7 @@ GIT_COMMIT_ID ?= $(shell git rev-parse --short=8 HEAD)
 OPERATOR_REGISTRY ?= quay.io
 OPERATOR_REPO_REF ?= $(OPERATOR_REGISTRY)/redhat-developer/servicebinding-operator
 OPERATOR_IMAGE_REF ?= $(OPERATOR_REPO_REF):$(GIT_COMMIT_ID)
+OPERATOR_IMAGE_SHA_REF ?= $(shell $(CONTAINER_RUNTIME) inspect --format='{{index .RepoDigests 0}}' $(OPERATOR_IMAGE_REF))
```

Comment:
Are we sure when this line gets evaluated? I mean, when the `docker inspect...` command is actually executed?
The thing is that the image SHA (a.k.a RepoDigest) is created/generated/assigned to the image only after pushing it into the repository. That is why the shell command to retrieve the digest was placed in-line.

In case the suggested line is actually evaluated before the image is actually pushed to the repository, it would be empty.

Reply:
it is a lazy evaluation, happens when the var is referenced, see: https://www.gnu.org/software/make/manual/make.html#Flavors

Reply:
Ok, cool!